weekend-bowlling/README.md

```diff
@@ -0,0 +1,50 @@
+# Weekend Challenge: Bowling Score Card
+Following the weekend challenge: [Bowling Challenge] (https://github.com/makersacademy/bowling-challenge)
```

Comment:
Removing the space between the ] and ( should mean the link is displayed properly